graph-builder/src/graph.rs

```diff
@@ -45,6 +46,11 @@ lazy_static! {
         "Total number of upstream scraping errors"
     )
     .unwrap();
+    static ref UPSTREAM_ERRORS_SUBSEQUENT: IntGauge = register_int_gauge!(
```

Comment:
That word fits much better! I removed the metric altogether though as per your next comment. ;-) I was aware that this can be calculated in a range query but I thought it might be a nice instant metric to have.